docs/04-guides/on-demand/store-video-asset-on-ipfs/_tabs.mdx

```diff
@@ -0,0 +1,36 @@
+import TabbedPages from "@site/src/components/TabbedPages";
+
+<head>
+  <title>Store Video Asset on IPFS | Livepeer Studio Docs</title>
+  <meta
+    property="og:title"
+    content="Store Video Asset on IPFS | Livepeer Studio Docs"
+  />
+</head>
+
+# Store Video Asset on IPFS
+
+This guide provides you the instructions to store a video asset on IPFS from the [Livepeer Studio Dashboard](./dashboard) 
+or through the [Livepeer API](./api).
+
+## Technical Preparation
+
+Before getting started, you will need to meet the following dependencies for 
+working with the On Demand feature:
+
+- [A Livepeer Studio account](https://livepeer.studio/login)
+- Install [Node.js](https://nodejs.org/en/)
+- Access to the [API Reference](https://docs.livepeer.studio/references)
+- An [API Key](https://docs.livepeer.studio/guides/livestreaming/api-key) to call the VOD API
+    - Note: Calling the API from the frontend requires an API key with CORS access allowed
+
+Throughout this guide you'll see a few terms mentioned multiple times, asset and task:
+- **Asset**: an object that represents a file
+- **Task**: an action taken on the file
```

Comment:
I had a hard time finding the API guide/tabs. I personally skip most of these intro sections unless there's something calling my attention like a "box"/notice or something like that. Here we have a lot of text that occupies half of the screen and then my mind just skipped to the `# Step 1` which was the most evident thing.  We could make those step headers smaller as well so they don't call the attention so much.

WDYT of removing this whole `Technical Preparation` section to be more straightforward? Can probably sum everything up in 1 short paragraph with a couple of links to other docs. e.g.
```suggestion
This guide provides you the instructions to store a video asset on IPFS from the [Livepeer Studio Dashboard](./dashboard) 
or through the [Livepeer API](./api). You should be familiar with [Assets](/reference/api/get-assets)
and [Tasks](/reference/api/get-tasks) and have a [Livepeer dev environment setup](/quickstart/)
```

(Later we should have specific docs explaining what are assets and tasks, just like we had for streams previously (tho I can't find it on the new docs)).

Reply:
Great points! I agree that the intro part is too long, will add your suggestion